.github/workflows/pr.yaml

```diff
@@ -0,0 +1,76 @@
+name: Pull-Request
+
+# Note that push and pull-request builds are automatically
+# now skipped by GitHub if
+# [skip ci], [ci skip], [no ci], [skip actions], or [actions skip]
+# are in the commit message. We don't need to check for this ourselves.
+
+on:
+  pull_request:
+    branches: [devel]
```

Comment:
Do we not want to check builds for fix PRs against `main` too? I doubt `main` and `devel` would be so different that the application of the fix would be drastically different between the two branches, but I could imagine cases where something might be subtly different if an new feature has already been applied on `devel` before the fix.

Reply:
Yes - you are right that I missed this ;-)